homeassistant/components/binary_sensor/alarmdecoder.py

```diff
@@ -75,6 +89,14 @@ def should_poll(self):
         """No polling needed."""
         return False
 
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes."""
+        attr = {}
+        if self._rfid:
+            attr[ATTR_RF_STATE] = self._rfstate
```

Comment:
What all information does this expose? If this is a dictionary we should be spreading it onto the state attributes using known keys, rather than nesting it all under one key.